.github/workflows/wheels.yaml

```
@@ -40,4 +42,33 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           name: wheels
-          path: ./wheelhouse/*.whl
+          path: ./dist/*.whl
+
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build sdist
+        run: pipx run build --sdist
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: dist/*.tar.gz
```

Not sure if this is necessary, but in yt_astro_analysis we gave this an explicit name that we used for the download action.
```suggestion
          name: tarball
          path: dist/*.tar.gz
```